client/my-sites/plugins/education-footer/index.tsx

```diff
@@ -0,0 +1,101 @@
+import styled from '@emotion/styled';
+import { useTranslate } from 'i18n-calypso';
+import FooterSection from 'calypso/components/footer-section';
+import LinkCard from 'calypso/components/link-card'; // should we host this one inside footer-section?
+
+interface Article {
+	title: string;
+	category: string;
+	url: string;
+	background: string;
+}
+
+const deslugify = ( string: string ) => string.replaceAll( '-', ' ' );
+const slugify = ( string: string ) =>
+	string
+		.replace( /[&/\\#,+()$~%.'":*?<>{}’]/g, '' )
+		.toLowerCase()
+		.replaceAll( ' ', '-' );
+
+const ArticleLinksContainer = styled.div`
+	@media ( max-width: 960px ) {
+		grid-template-columns: repeat( 1, 1fr );
+	}
+
+	@media ( max-width: 660px ) {
+		padding: 0 16px;
+	}
+
+	display: grid;
+	grid-template-columns: repeat( 3, 1fr );
+	column-gap: 29px;
+	row-gap: 10px;
+`;
+
+const EducationFooter = () => {
+	const translate = useTranslate();
+
+	const articles: Article[] = [
```

Comment:
Do we need this array? Is this data going to come from an endpoint at somepoint? Up to you but it might make sense to just inline all this into repeated <LinkCard> jsx.

Reply:
Probably not, just me preparing for an scenario where we receive this from an API

Reply:
✅